art/estimators/object_detection/pytorch_object_detector.py

```diff
@@ -247,8 +249,41 @@ def _preprocess_and_convert_inputs(
 
         return x_preprocessed, y_preprocessed
 
+    def _translate_labels(self, labels: List[Dict[str, "torch.Tensor"]]) -> Any:
```

Comment:
Can we define the return type more accurately?

Reply:
Since `_translate_labels()` and `_translate_predictions()` need to be overridden for each object detector model, this can take many different types (e.g., dictionary for FRCNN, tensor for Yolo, list of tensor for Detr). Replacing the `Any` would require a very long Union with every possible type for all subtypes which would need to updated if any new object detector is added. Therefore, it makes the most sense to keep this an `Any` type.